docs/Getting Started/Installation/index.md

```diff
@@ -3,7 +3,24 @@ In order to build your app on a SmartDeviceLink (SDL) Core, the SDL software dev
 
 ## Install SDL SDK
 @![iOS]
-There are three different ways to install the SDL SDK in your project: CocoaPods, Carthage, or manually.
+There are four different ways to install the SDL SDK in your project: Accio, CocoaPods, Carthage, or manually.
+
+### Accio
```

Comment:
Please put this below the CocoaPods section

Reply:
moved